PhysicsTools/PatAlgos/python/producersLayer1/lowPtElectronProducer_cff.py

```diff
@@ -67,22 +67,14 @@
                                                            genParticleMatch = "electronMatch"
                                                            )
 
-# For run2_miniAOD_UL ...
+# For (~bParking & run2_miniAOD_UL): (1) rekey, (2) rerun ID, (3) apply energy regression
 from Configuration.ProcessModifiers.run2_miniAOD_UL_cff import run2_miniAOD_UL
-_makePatLowPtElectronsTask = makePatLowPtElectronsTask.copy()
-
-# (1) rekey seed BDT ValueMaps by reco::GsfElectron
+from Configuration.Eras.Modifier_bParking_cff import bParking
 from RecoEgamma.EgammaElectronProducers.lowPtGsfElectronSeedValueMaps_cff import rekeyLowPtGsfElectronSeedValueMaps
-_makePatLowPtElectronsTask.add(rekeyLowPtGsfElectronSeedValueMaps)
-
-# (2) rerun ID
 from RecoEgamma.EgammaElectronProducers.lowPtGsfElectronID_cfi import lowPtGsfElectronID
-_makePatLowPtElectronsTask.add(lowPtGsfElectronID)
-
-# (3) apply energy regression
 from RecoEgamma.EgammaElectronProducers.lowPtGsfElectrons_cfi import lowPtGsfElectrons
+_makePatLowPtElectronsTask = makePatLowPtElectronsTask.copy()
+_makePatLowPtElectronsTask.add(rekeyLowPtGsfElectronSeedValueMaps)
+_makePatLowPtElectronsTask.add(lowPtGsfElectronID)
 _makePatLowPtElectronsTask.add(lowPtGsfElectrons)
-
-# Append to Task
-run2_miniAOD_UL.toReplaceWith(makePatLowPtElectronsTask,_makePatLowPtElectronsTask)
-
+(~bParking & run2_miniAOD_UL).toReplaceWith(makePatLowPtElectronsTask,_makePatLowPtElectronsTask)
```

Comment:
it looks like this update assumes that bParking UL AOD is made with a correct ID.
I guess this reconfiguration is perhaps pending resolution of actually deploying LowPtElectrons/LowPtElectrons_ID_2021May17.root in the UL bParking production

Reply:
If I understand you correctly, you're saying that the logic above implicitly assumes the the BParking UL RE-RECO will use May17, and so nothing is required on the part of re-miniAOD (to e.g. update the ID). 

If so, you are correct and I am updating the back port now with the logic for (bParking & run2_miniAOD_UL) that will indeed rerun the ID (May17). This change will be added to this PR.